### PR TITLE
Droxx keymap

### PR DESCRIPTION
**Pull request by furfuzz**
## Description
A kymap for the KBD67 Rev 2 pcb

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd67/rev2/keymaps/droxx/config.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#pragma once
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/rev2/keymaps/droxx/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/kbdfans/kbd67/rev2/keymaps/droxx/rules.mk [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+COMMAND_ENABLE = no        	# Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no      	# Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes       # Enable keyboard underglow functionality
+MIDI_ENABLE = no            # MIDI controls (+2400 to 4200)
+AUDIO_ENABLE = no           # Enable the audio subsystem
+BLUETOOTH_ENABLE = no		# Enable Bluetooth with the Adafruit EZ-Key HID
+
+# Do not enable SLEEP_LED_ENABLE. It uses the same timer as BACKLIGHT_ENABLE.
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/rev2/keymaps/droxx/rules.mk:17]: This really should only be what is different. 
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
BACKLIGHT_ENABLE = no      	# Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard underglow functionality
```

File: keyboards/kbdfans/kbd67/rev2/keymaps/droxx/config.h [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+/* force N key rollover even on startup regardless of EEPROM setting */
+#define FORCE_NKRO
+
+/* USB polling rate */
+#define USB_POLLING_INTERVAL_MS 1
+
+/* turn off RGB when computer sleeps */
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_SLEEP
+#endif
```
Comment (drashna) [inline keyboards/kbdfans/kbd67/rev2/keymaps/droxx/config.h:51]: Most of this is already defined, and should not be included here.  Only the values that are different should be included.
```suggestion
#define FORCE_NKRO
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by furfuzz — COMMENTED** [review]
addressed suggestions

**Review by drashna — APPROVED** [review]

File: keyboards/kbdfans/kbd67/rev2/keymaps/droxx/rules.mk [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Build Options
+#   change yes to no to disable
+#
+MOUSEKEY_ENABLE = no        # Mouse keys (+4700)
+BACKLIGHT_ENABLE = no      	# Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes       # Enable keyboard underglow functionality
```
Comment (fauxpark) [inline keyboards/kbdfans/kbd67/rev2/keymaps/droxx/rules.mk:6]:
```suggestion
MOUSEKEY_ENABLE = no
BACKLIGHT_ENABLE = no
RGBLIGHT_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by furfuzz — COMMENTED** [review]
addressed rules.mk suggestions

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**